tests/test_views.py

```diff
@@ -225,3 +225,10 @@ def test_post__empty(self) -> None:
         assert [] == form.non_field_errors()
         assert {'body': ['This field is required.']} == form.errors
         assert not form.is_valid()
+
+    def test_post__valid(self) -> None:
+        self.client.force_login(self.user)
+        response: HttpResponse = self.client.post(self.path, data={
+            'body': 'An example answer',
+        })
+        self.assertRedirects(response, 'TODO')
```

Comment:
@sewagodimo: I added my last bits from the hackathon tonight, up to this point: do you want to tackle the rest on `AnswerCreate`?

Reply:
Yes I want to give it a shot, thanks @pjdelport